packages/ui-react/src/components/LoginForm/LoginForm.tsx

```diff
@@ -45,8 +44,8 @@ const LoginForm: React.FC<Props> = ({ onSubmit, onChange, socialLoginURLs, value
           {errors.form}
         </FormFeedback>
       ) : null}
-
-      <SocialButtonsList socialLoginURLs={socialLoginURLs} />
+      {/* Social login feature is currently not in use, so the SocialButtonsList component is commented out for now. */}
+      {/* <SocialButtonsList socialLoginURLs={socialLoginURLs} /> */}
```

Comment:
Is there a strong reason to comment this out instead of just removing it? If there is no set plan to re-enable it in the short term, I would suggest we remove it entirely. We can always retrieve it from git history.

Reply:
07302ef480de81151c98e648bfa555126c496ef1